modules/ROOT/pages/update-modules.adoc

```diff
@@ -1,24 +1,21 @@
 = Update Your Modules
 
-Every time Studio starts up it checks for updated modules in your open projects. You can choose to update to the latest minor version of your module or upgrade directly to the latest version available.
+Every time Studio starts up it checks for updated modules in your open projects. You can choose to update to the latest minor version of your module or upgrade directly to the latest version available. Upgrading to a major version of a module can introduce compatibility issues in your project. Review the release notes of your modules before upgrading to a major version.
 
-[NOTE]
-Consider that upgrading to a major version of a module might introduce compatibility issues in your project. Review the release notes of your modules before upgrading major versions.
+NOTE: If you xref:change-jdk-config-in-projects.adoc[select Java 17 for your project], Studio can notify you about updating your modules to make them compatible with the selected Java version.
 
-To manually check for modules updates:
+To manually check for module updates:
 
 . Ensure that you are logged in to Anypoint Platform.
-. In the top menu bar, select the *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]) button.
+. In the menu bar, select the *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]) button.
```

Comment:
```suggestion
. In the menu bar, click *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]).
```

Reply:
Sorry! I read it but I didn't see it! :/